config/application.rb

```diff
@@ -62,7 +62,6 @@ class DavidRunger::Application < Rails::Application
     g.factory_bot(dir: 'spec/factories')
   end
 
-  config.middleware.insert_after(ActionDispatch::Static, Rack::Deflater) # gzip all responses
```

Comment:
There's no longer a need for this, since NGINX will compress the assets with gzip. It also will no longer work, since Rails won't add the `ActionDispatch::Static` middleware.